dox/books/Primer/main_pdf.dox

```diff
@@ -27,12 +27,10 @@ It is a general purpose statically typed, easy to learn and use language.
 - ease of static typing
 - strong static typing
 
-@htmlonly
 <h4>License</h4>
```

Comment:
I should have deleted these. In latex, the License notification is in the latex header file.

Please remove the license from this file. (In the reference manual, main_pdf.dox too.)

Reply:
Removed from dox/books/*/main_pdf.dox